profiles/package.mask

```diff
@@ -1,3 +1,12 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-10)
+# The older versions of these packages do not compile with some of the newer
+# dependencies that have been added.
+<dev-haskell/persistent-test-2.0.3.5
```

Comment:
Having build failures for older failures is natural. We usually just delete old package versions. Not mask them.